pkg/monitor/intervalcreation/node.go

```diff
@@ -191,6 +200,46 @@ func eventsFromKubeletLogs(nodeName string, kubeletLog []byte) monitorapi.Interv
 	return ret
 }
 
+// eventsFromOVSVswitchdLogs returns the produced intervals.  Any errors during this creation are logged, but
+// not returned because this is a best effort step
+func eventsFromOVSVswitchdLogs(nodeName string, ovsLogs []byte) monitorapi.Intervals {
+	nodeLocator := monitorapi.NodeLocator(nodeName)
+	ret := monitorapi.Intervals{}
+
+	scanner := bufio.NewScanner(bytes.NewBuffer(ovsLogs))
+	for scanner.Scan() {
+		currLine := scanner.Text()
+		ret = append(ret, unreasonablyLongPollInterval(currLine, nodeLocator)...)
+	}
+
+	return ret
+}
+
+// unreasonablyLongPollInterval searches for a failure associated with https://issues.redhat.com/browse/OCPBUGS-11591
+//
+// Apr 12 11:53:51.395838 ci-op-xs3rnrtc-2d4c7-4mhm7-worker-b-dwc7w ovs-vswitchd[1124]:
+//
+//	ovs|00002|timeval(urcu4)|WARN|Unreasonably long 109127ms poll interval (0ms user, 0ms system)
+func unreasonablyLongPollInterval(logLine, nodeLocator string) monitorapi.Intervals {
+	if !strings.Contains(logLine, "Unreasonably long") {
+		return nil
+	}
+
+	failureTime := systemdJournalLogTime(logLine)
+	message := logLine[strings.Index(logLine, "ovs-vswitchd"):]
+	return monitorapi.Intervals{
+		{
+			Condition: monitorapi.Condition{
+				Level:   monitorapi.Warning,
```

Comment:
Does this needs to be `monitorapi.Error` ?

Reply:
I went with warning as that was what the author of the log line chose, I'm not sure if there's a case to call it an error here in and of itself.

Reply:
Fair enough, lets go with what the author chose.